setup.py

```diff
@@ -1,55 +1,65 @@
-from setuptools import setup, find_packages
-
 import codecs
 import os
 import re
-import sys
+
+from setuptools import setup, find_packages
 
 here = os.path.abspath(os.path.dirname(__file__))
 
 
 min_requires = [
-    'google-auth-httplib2',
-    'google-auth',
-    'google-cloud-core>=1.4.1',
-    'google-cloud-storage',
-    'requests',
-    'tqdm',
-    'urllib3',
-    'deprecated',
-    'python-dotenv',
-    'gcsfs>=0.3.0,<0.7',
-    "retry"
+    "deprecated",
+    "gcsfs>=0.3.0,<0.7",
+    "google-auth",
+    "google-auth-httplib2",
+    "google-cloud-bigquery",
+    "google-cloud-core>=1.4.1",
+    "google-cloud-storage",
+    "joblib>=0.11",
+    "numpy>=1.16.3",
+    "pandas>=0.23.4,!=1.0.4",
+    "pyarrow>=0.15.1,<1.0.0",
+    "python-dotenv",
+    "requests",
+    "retry",
+    "tqdm",
+    "urllib3",
 ]
 
 extras_require = {
-    "dataframe": min_requires + ['pandas>=0.23.4,!=1.0.4', 'numpy>=1.16.3', 'joblib>=0.11', 'pyarrow>=0.15.1,<1.0.0',],
-    "pipeline": min_requires + ['luigi', 'papermill', 'pandas>=0.23.4,!=1.0.4', 'numpy>=1.16.3', 'joblib>=0.11',
-                                'pyarrow>=0.15.1,<1.0.0',],
-    "onlineapp": min_requires + ['flask>=1.0.2', 'redis'],
-    "dask": min_requires + ['dask[complete]'],
-    "dev": min_requires + ['pytest', 'bumpversion', "sphinx-rtd-theme", "sphinx"],
+    "pipeline": min_requires
+    + [
+        "luigi",
+        "papermill",
+        "pandas>=0.23.4,!=1.0.4",
```

Comment:
if we maintain pandas in the min req. we don't need to have it here too. 
Also, we will need to update the README.md